src/ucp/core/ucp_worker.h

```diff
@@ -35,6 +35,15 @@ enum {
 
 
 /**
+ * UCP worker flags
+ */
+enum {
+    UCP_WORKER_FLAG_EXTERNAL_EVENT_FD = UCS_BIT(0), /**< epoll fd is external */
+    UCP_WORKER_FLAG_EDGE_TRIGGERED    = UCS_BIT(1)  /**< events are edge-triggered */
```

Comment:
do we define anywhere what is edge event ?

Reply:
added explanation in api/ucp.h